src/main/java/TED.java

```diff
@@ -0,0 +1,98 @@
+import java.util.Scanner;
+
+
+
+public class TED {
+    public static void main(String[] args) {
+        String line;
+        String elem;
```

Comment:
Consider spelling out the whole word 'elem'

Reply:
what does elem mean?